Resources/public/css/theme/views/testonearticles.css

```diff
@@ -0,0 +1,26 @@
+.ez-view-testonearticlesview .article__list {
+    list-style: none;
+}
+
+.ez-view-testonearticlesview .article__link {
+    cursor: pointer;
+}
+
+.ez-view-testonearticlesview .article__link--active {
+    background-color: lightgray;
+}
+
+.ez-view-testonearticlesview.ez-view-testonearticlesview--loading .ez-inline-loader {
+    display: block;
+}
+
+.ez-view-testonearticlesview.ez-view-testonearticlesview--loading li .ez-inline-loader {
+    display: inline-block;
+    margin-left: 10px;
```

Comment:
The same as: https://github.com/gbalcewicz/PlatformUIBundle/pull/1/files#r41961151